core/modules/simpletest/tests/form_test.module

```diff
@@ -1716,6 +1722,45 @@ function form_test_h5datetime_submit($form, &$form_state) {
   exit();
 }
 
+/**
+ * Form constructor for disabled html_date and html_time elements.
+ *
+ * @see form_test_disabled_h5date_h5time_submit()
+ */
+function form_test_disabled_h5date_h5time($form, &$form_state) {
+  $form['disabled_h5date'] = array(
```

Comment:
Everywhere we have `disabled` in a form element or function name, could you rename these to `no_access`? So either `form_test_no_access_h5date_h5time` or `form_test_h5date_h5time_no_access` for the function names and `no_access_h5date` or `h5date_no_access` for the form elements.

The reason being that setting `#disabled` is not the same as `#access = FALSE`. It might actually be good test both scenarios here if that isn't too much additional work.

Reply:
@quicksketch, the problem exists for both `'#access' => FALSE` AND for `'#disabled' => TRUE'` cases. I'll switch to `no_access` as requested for the function name, and will see if I can add a test case for the disabled one.